pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -488,14 +488,14 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *promv1.Prome
 				Name: "kubevirt.rules",
 				Rules: []promv1.Rule{
 					{
-						Record: "num_of_running_virt_api_servers",
```

Comment:
This PR is interchanging the word `running` with `up`. I want to make sure this verbage is the direction we should be going in or not.

In general, the term `running` means that a pod is in phase=Running, and the term `ready` means that the pod is both running and has initialized to the point the readiness checks are passing. So, to me, sticking with `running` is more intuitive from a k8s naming convention standpoint `up`. I'd be able to accurately infer what `kubevirt_running_virt_api_servers_total` means fairly accurately.

Reply:
@davidvossel The metric is based on the Prometheus built in "up" metric and we filter by the namespace and pod. The question is if this metric is enough to define that the operator is running?

Reply:
where is this `up` metric documented? I can't find anything signifying what it does.